src/main/java/model/students/StudentsList.java

```diff
@@ -1,23 +1,34 @@
 package model.students;
 
+import model.exceptions.NoContactDataException;
+import model.exceptions.NotValidEmailException;
+import model.exceptions.NotValidPhoneException;
+
+import java.io.IOException;
+import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
+import static util.Validator.checkContactData;
+import static util.Writer.writeToFile;
+
 /**
  * Created by aleksey.dobrovolsky on 6/3/2019.
  */
-class StudentsList {
+public class StudentsList {
```

Comment:
There are too many things StudentsList can do. It knows how to parse students list from file, how to load the file.
You should either change class name to StudentsFileLoader or split functionality of the class. Try to select one target for the class so there is only 1 reason to change the class